README.md

```diff
@@ -4,39 +4,53 @@ This repository generates the documentation available at https://docs.mattermost
+
+If this is your first time contributing to Mattermost, first read and sign the [Mattermost Contributor Agreement](https://www.mattermost.org/mattermost-contributor-agreement/) and sign it (at the bottom of the page), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true).
+
+
+**Editing** 
+
+The quickest way to begin is editing directly on GitHub, on your fork of the Mattermost docs repo. 
+
+Click the **Edit** icon on the top right corner of the page you want to edit in the Mattermost documentation.
+
+If this the first time you're contributing, click **Edit the file in your fork of this project** (pencil icon) on the top right corner to create a fork. 
+
+## Creating Pull Requests
+
+1. When you're ready to submit your changes, add a descriptive title and comments to summarize the changes made.
+2. Select **Create a new branch for this commit and start a pull request**.
+3. Check the **Propose file change** button.
+4. Scroll down to compare changes with the original document.
+5. Select **Create pull request**. 
 
-## Contributing
 
-**How to create Pull Requests to edit or create Mattermost documentation (by community members):**
+## Building and Validating
 
-1. Sign the Contributor License Agreement (see instructions in the next section).
-3. On the Mattermost Documentation page that you want to edit, click the GitHub icon on the upper right corner that says "Edit".
-4. Click "Edit the file in your fork of this project" (pencil icon) on the upper right corner.
-5. After making changes, check the "Propose file change" button.
-6. Compare changes with the original document.
-7. Click "Create a Pull Request". Make sure that the Pull Request has a descriptive title and add comments to briefly tell others what you have worked on (optional).
+If you're editing Mattermost documentation on a local machine, you can generate the HTML files from markdown in the `/source` directory to review them before you commit changes or create pull requests. 
 
-**Signing CLA:**
+1. Download the Mattermost docs repo onto a machine with Python installed.
+2. Install [pipenv](https://docs.pipenv.org/): `pip install pipenv`. 
+3. `cd` into the cloned repository.
+4. Install the required packages: `pipenv install`.
+5. Build: `make html` - generates files in `/build` directory.
 
-Please read the [Mattermost Contributor Agreement](https://www.mattermost.org/mattermost-contributor-agreement/) and sign it (at the bottom of the page), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true).
 
-**Additional process for Core Committers for Doc Repo:**
+**Additional Process for Core Committers**
```

Comment:
```suggestion
**Additional Process for Core Committers**
```
I'm not really sure whether the steps below are the process, or whether there's a process that's not mentioned? I renamed the label references based on the labels in GitHub.

Reply:
This is a regular process for the person who owns Editor Review label. I think it should be more of a checklist than a step-by-step process.

Reply:
Do we have a document with a workflow for editor review anywhere? It might be best to move this to that type of document seeing as this page is guidelines for contributing and contributors don't generally edit documentation (do they?). 

Might be helpful to have a general process flow and task list, and then add this information to that instead - if we don't have one already?